### ISSUE TITLE
2.0.0-M.5 - FreeMarker template issue with sub-variable "referencedNamespaces"

### ISSUE THREAD

**Issue by scoquelin** [stoicflame/enunciate#148]
My enunciate.xml : 

````
<?xml version="1.0" encoding="UTF-8"?>
<enunciate xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
	xsi:noNamespaceSchemaLocation="http://enunciate.webcohesion.com/schemas/enunciate-2.0.0-M.5.xsd">
	<modules>
		<spring disabled="true" />
		<java-json-client disabled="true" />
		<java-xml-client disabled="true" />
	</modules>
</enunciate>
````
Here is the stacktrace that I get, let me know if you need more info.

````
[DEBUG] [ENUNCIATE] Default base to be used for documentation base.
[DEBUG] [ENUNCIATE] Extracting css/ to /home/myproject/css.
[DEBUG] [ENUNCIATE] Extracting css/style.css to /home/myproject/css/style.css.
Sep 17, 2015 12:42:51 PM freemarker.log._JULLoggerFactory$JULLogger error
SEVERE: Error executing FreeMarker template
FreeMarker template error:
An error has occurred when reading existing sub-variable "referencedNamespaces"; see cause exception! The type of the containing value was: extended_hash+string (com.webcohesion.enunciate.modules.jaxb.model.SchemaInfo wrapped into f.e.b.StringModel)

----
FTL stack trace ("~" means nesting-related):
	- Failed at: #list schema.referencedNamespaces as ...  [in template "jar:file:/home/seb/.m2/repository/com/webcohesion/enunciate/enunciate-idl/2.0.0-M.5/enunciate-idl-2.0.0-M.5.jar!/com/webcohesion/enunciate/modules/idl/schema.fmt" at line 10, column 3]
----

Java stack trace (for programmers):
----
freemarker.core._TemplateModelException: [... Exception message was already printed; see it above ...]
	at freemarker.ext.beans.BeanModel.get(BeanModel.java:199)
	at freemarker.core.Dot._eval(Dot.java:40)
	at freemarker.core.Expression.eval(Expression.java:78)
	at freemarker.core.IteratorBlock.accept(IteratorBlock.java:58)
	at freemarker.core.Environment.visit(Environment.java:312)
	at freemarker.core.MixedContent.accept(MixedContent.java:62)
	at freemarker.core.Environment.visit(Environment.java:312)
	at freemarker.core.Environment.process(Environment.java:290)
	at freemarker.template.Template.process(Template.java:312)
	at com.webcohesion.enunciate.modules.idl.BaseXMLInterfaceDescriptionFile.processTemplate(BaseXMLInterfaceDescriptionFile.java:106)
	at com.webcohesion.enunciate.modules.idl.BaseXMLInterfaceDescriptionFile.writeTo(BaseXMLInterfaceDescriptionFile.java:60)
	at com.webcohesion.enunciate.modules.idl.BaseXMLInterfaceDescriptionFile.writeTo(BaseXMLInterfaceDescriptionFile.java:48)
	at com.webcohesion.enunciate.modules.docs.DocsModule.call(DocsModule.java:259)
	at com.webcohesion.enunciate.io.InvokeEnunciateModule.onNext(InvokeEnunciateModule.java:31)
	at com.webcohesion.enunciate.io.InvokeEnunciateModule.onNext(InvokeEnunciateModule.java:10)
	at rx.internal.operators.OperatorDoOnEach$1.onNext(OperatorDoOnEach.java:79)
	at rx.internal.operators.OperatorSingle$ParentSubscriber.onCompleted(OperatorSingle.java:109)
	at rx.internal.operators.NotificationLite.accept(NotificationLite.java:140)
	at rx.internal.operators.TakeLastQueueProducer.emit(TakeLastQueueProducer.java:100)
	at rx.internal.operators.TakeLastQueueProducer.startEmitting(TakeLastQueueProducer.java:45)
	at rx.internal.operators.OperatorTakeLast$1.onCompleted(OperatorTakeLast.java:59)
	at rx.internal.operators.OperatorMerge$MergeSubscriber.drainAndComplete(OperatorMerge.java:521)
	at rx.internal.operators.OperatorMerge$MergeSubscriber.completeInner(OperatorMerge.java:486)
	at rx.internal.operators.OperatorMerge$InnerSubscriber.emit(OperatorMerge.java:658)
	at rx.internal.operators.OperatorMerge$InnerSubscriber.onCompleted(OperatorMerge.java:600)
	at rx.subjects.SubjectSubscriptionManager$SubjectObserver.onCompleted(SubjectSubscriptionManager.java:232)
	at rx.subjects.ReplaySubject.onCompleted(ReplaySubject.java:404)
	at rx.Observable$35.onCompleted(Observable.java:7458)
	at rx.observers.SafeSubscriber.onCompleted(SafeSubscriber.java:81)
	at rx.internal.operators.OperatorDoOnEach$1.onCompleted(OperatorDoOnEach.java:53)
	at rx.internal.operators.OperatorSingle$ParentSubscriber.onCompleted(OperatorSingle.java:110)
	at rx.internal.operators.NotificationLite.accept(NotificationLite.java:140)
	at rx.internal.operators.TakeLastQueueProducer.emit(TakeLastQueueProducer.java:100)
	at rx.internal.operators.TakeLastQueueProducer.startEmitting(TakeLastQueueProducer.java:45)
	at rx.internal.operators.OperatorTakeLast$1.onCompleted(OperatorTakeLast.java:59)
	at rx.internal.operators.OperatorMerge$MergeSubscriber.drainAndComplete(OperatorMerge.java:521)
	at rx.internal.operators.OperatorMerge$MergeSubscriber.completeInner(OperatorMerge.java:486)
	at rx.internal.operators.OperatorMerge$InnerSubscriber.emit(OperatorMerge.java:658)
	at rx.internal.operators.OperatorMerge$InnerSubscriber.onCompleted(OperatorMerge.java:600)
	at rx.internal.operators.OperatorMerge$MergeSubscriber.drainAndComplete(OperatorMerge.java:521)
	at rx.internal.operators.OperatorMerge$MergeSubscriber.completeInner(OperatorMerge.java:486)
	at rx.internal.operators.OperatorMerge$InnerSubscriber.emit(OperatorMerge.java:673)
	at rx.internal.operators.OperatorMerge$InnerSubscriber.onCompleted(OperatorMerge.java:600)
	at rx.internal.operators.OperatorMerge$MergeSubscriber.drainAndComplete(OperatorMerge.java:521)
	at rx.internal.operators.OperatorMerge$MergeSubscriber.completeInner(OperatorMerge.java:486)
	at rx.internal.operators.OperatorMerge$InnerSubscriber.emit(OperatorMerge.java:673)
	at rx.internal.operators.OperatorMerge$InnerSubscriber.onCompleted(OperatorMerge.java:600)
	at rx.internal.operators.OperatorMerge$MergeSubscriber.drainAndComplete(OperatorMerge.java:521)
	at rx.internal.operators.OperatorMerge$MergeSubscriber.completeInner(OperatorMerge.java:486)
	at rx.internal.operators.OperatorMerge$InnerSubscriber.emit(OperatorMerge.java:673)
	at rx.internal.operators.OperatorMerge$InnerSubscriber.onCompleted(OperatorMerge.java:600)
	at rx.internal.util.ScalarSynchronousObservable$ScalarSynchronousAction.call(ScalarSynchronousObservable.java:117)
	at rx.schedulers.ExecutorScheduler$ExecutorAction.run(ExecutorScheduler.java:173)
	at rx.schedulers.ExecutorScheduler$ExecutorSchedulerWorker.run(ExecutorScheduler.java:99)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at freemarker.ext.beans.BeansWrapper.invokeMethod(BeansWrapper.java:1458)
	at freemarker.ext.beans.BeanModel.invokeThroughDescriptor(BeanModel.java:255)
	at freemarker.ext.beans.BeanModel.get(BeanModel.java:159)
	... 56 more
Caused by: com.webcohesion.enunciate.EnunciateException: none is not recognized as an XML type.
	at com.webcohesion.enunciate.modules.jaxb.model.types.XmlTypeVisitor.defaultAction(XmlTypeVisitor.java:39)
	at com.webcohesion.enunciate.modules.jaxb.model.types.XmlTypeVisitor.defaultAction(XmlTypeVisitor.java:35)
	at javax.lang.model.util.SimpleTypeVisitor6.visitNoType(SimpleTypeVisitor6.java:226)
	at com.webcohesion.enunciate.javac.decorations.type.DecoratedNoType.accept(DecoratedNoType.java:37)
	at com.webcohesion.enunciate.modules.jaxb.model.types.XmlTypeFactory.getXmlType(XmlTypeFactory.java:163)
	at com.webcohesion.enunciate.modules.jaxb.model.ComplexTypeDefinition.getBaseType(ComplexTypeDefinition.java:42)
	at com.webcohesion.enunciate.modules.jaxb.model.SchemaInfo.addReferencedNamespaces(SchemaInfo.java:358)
	at com.webcohesion.enunciate.modules.jaxb.model.SchemaInfo.addReferencedNamespaces(SchemaInfo.java:312)
	at com.webcohesion.enunciate.modules.jaxb.model.SchemaInfo.getReferencedNamespaces(SchemaInfo.java:258)
	... 63 more
````

**Comment by stoicflame** [comment]
Fixed at b4dd0f3.

**Comment by stoicflame** [comment]
Please confirm the fix with [Release Candidate 1](https://github.com/stoicflame/enunciate/releases/tag/v2.0.0-RC.1)
